cobalt/BUILD.gn

```diff
@@ -80,18 +80,18 @@ action("cobalt_build_info") {
   ]
 }
 
-if (is_android) {
-  test("cobalt_unittests") {
-    testonly = true
+test("cobalt_unittests") {
+  testonly = true
 
-    sources = [ "//cobalt/user_agent/user_agent_platform_info_test.cc" ]
+  sources = [
```

Comment:
This looks like it's formatted incorrectly

Reply:
Ah yes, I did it by hand and took a bit of another CL that is blocked by this.